src/__tests__/Search/CompareWithBase.test.tsx

```diff
@@ -23,7 +23,7 @@ function setUpTestData() {
     })
     .get('begin:https://treeherder.mozilla.org/api/perfcompare/results/', [])
     .get(
-      'begin:https://treeherder.mozilla.org/api/project/mozilla-central/push/?revision=coconut',
+      'begin:https://treeherder.mozilla.org/api/project/try/push/?revision=coconut',
```

Comment:
I wasn't expecting some test change initially, but actually we had an inconsistency here: indeed the "coconut" revision in the test data is from the repository "try", not the repository "mozilla-central". So I had to change this test a little bit as well as the snapshots.